cmd/src/campaigns_common.go

```diff
@@ -100,6 +101,20 @@ func newCampaignsApplyFlags(flagSet *flag.FlagSet, cacheDir, tempDir string) *ca
 		"If true, errors encountered while executing steps in a repository won't stop the execution of the campaign spec but only cause that repository to be skipped.",
 	)
 
+	// We default to bind workspaces on everything except AMD64 macOS at
```

Comment:
```suggestion
	// We default to bind workspaces on everything except ARM64 macOS at
```

Right? Maybe we should use "macOS on M1 chips" as the wording 😄

Reply:
Ugh, thanks. I definitely did this a few other times (that I caught) when I was writing this.

I'm trying to avoid M1, since there will presumably be more CPUs in the future and we don't know Apple's naming scheme yet, but this is definitely a drawback to using the more technically correct name.